_playwright-tests/helpers/loginHelpers.ts

```diff
@@ -77,3 +82,23 @@ export const throwIfMissingEnvVariables = () => {
     throw new Error('Missing env variables:' + missing.join(','));
   }
 };
+
+export const ensureNotInPreview = async (page: Page) => {
+  const toggle = page.locator('.pf-v6-c-switch__toggle');
```

Comment:
This selector may be just waaay to broad. Any switch anywhere would satisfy this condition